packages/experimental/Drawer/src/Drawer.styling.ts

```diff
@@ -0,0 +1,54 @@
+import type { Theme, UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+
+import { DrawerName } from './Drawer.types';
+import type { DrawerTokens, DrawerSlotProps, DrawerProps } from './Drawer.types';
+import { defaultDrawerTokens } from './DrawerTokens';
+
+export const stylingSettings: UseStylingOptions<DrawerProps, DrawerSlotProps, DrawerTokens> = {
+  tokens: [defaultDrawerTokens, DrawerName],
+  tokensThatAreAlsoProps: 'all',
+  states: ['left', 'right', 'bottom', 'scrimVisible'],
+  slotProps: {
+    backdropContent: buildProps(
+      (tokens: DrawerTokens, _theme: Theme) => ({
+        style: {
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: '100%',
+          backgroundColor: tokens.backdropColor,
+          opacity: tokens.backdropOpacity,
+        },
+      }),
+      ['backdropColor'],
+    ),
+    content: buildProps(
+      (tokens: DrawerTokens, _theme: Theme) => ({
+        style: {
+          position: 'absolute',
+          height: tokens.height,
+          width: tokens.width,
+          elevation: tokens.drawerElevation,
+          ...(tokens.position === 'right' && { right: 0 }), // This is required to make the drawer stick to the right side of the screen as left overrides right position
```

Comment:
You shouldn't have to do this. Let DrawerTokens set the appropriate default tokens for each position state, and simply set right: tokens.[what you named the token] here

Reply:
Yes took. [token Name] approach.